doc/conf.py

```diff
@@ -56,26 +50,17 @@
 
 # General information about the project.
 project = 'dbxincluder'
-copyright = '2015, Thomas Schraitle'
-author = 'Thomas Schraitle'
+copyright = '2016, Fabian Vogt'
+author = 'Fabian Vogt'
```

Comment:
It's common sense in the open source world and considered as polite IMHO.
For example, Manuel and I did the same with docmanger (original code was from Rick). Although we rewrote most of his code, he is still mentioned as an author.